grafana/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BUILD_FROM=ghcr.io/hassio-addons/debian-base/amd64:6.0.0
+ARG BUILD_FROM=ghcr.io/hassio-addons/debian-base/amd64:6.1.0
```

Comment:
This should go into a separate PR, as it is not directly related to the Grafana upgrade.